### DISCUSSION TITLE
Consider flatpak

### DISCUSSION THREAD

**Discussion by ghost** [neovide/neovide#1194] [Ideas]
Snap surely is very popular among Debian derivatives' users. 
However creation of [flatpak](https://www.flatpak.org/) to be published on  [flathub](https://flathub.org/home) could be considered for easier distribution among RHEL derivatives' users & not only :)

**Comment by sedbytes** [comment]
It's in development currently.

Take a look at #634.

**Reply by ghost** [reply]
Thanks! Searching for "flatpak" returned zero issues for me somehow :3
